core/sql/executor/hiveHook.cpp

```diff
@@ -319,7 +319,7 @@ struct hive_sd_desc* populateSD(HiveMetaData *md, Int32 mainSdID,
   
   // TODO : loop over SDs
   if (findAToken(md, tblStr, pos, "sd:StorageDescriptor(", 
-                 "getTableDesc::sd:StorageDescriptor(###"))
+                 "getTableDesc::sd:StorageDescriptor(###"),FALSE)
```

Comment:
Is the idea to suppress the error and see if the hive operation succeeds ? All the other calls above this seem to default to TRUE?